test/remark-component.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Remark } from '../src';
+
+describe('Remark', () => {
```

Comment:
These tests focus on testing the React abstraction, remark-rehype and rehype-react have their own tests verifying content is converted as expected.